<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForcesTab</class>
 <widget class="QWidget" name="forcesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>413</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
   <item row="1" column="1">
    <spacer name="verticalSpacer_10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Force Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkForceRandomStartLocation">
        <property name="text">
         <string>Random Start Location</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="chkForceSharedVision">
        <property name="text">
         <string>Shared Vision</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkForceAlliedVictory">
        <property name="text">
         <string>Allied Victory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkForceAllies">
        <property name="text">
         <string>Allies</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="grpForceCustomName">
        <property name="title">
         <string>Custom Name</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="txtForceName"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QTreeWidget" name="forcesTree">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="indentation">
      <number>8</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>32</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>64</number>
     </attribute>
     <column>
      <property name="text">
       <string>Slot</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Race</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Colour</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Force 1</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Force 2</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Force 3</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Force 4</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
       </brush>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
